src/handles/handleDiagnostic.ts

```diff
@@ -150,8 +150,10 @@ async function handleLinter (
     command,
     rootPatterns = [],
     args = [],
-    offsetLine = 0,
-    offsetColumn = 0,
+    offsetLineStart = 0,
+    offsetLineEnd = 0,
+    offsetColumnStart = 0,
+    offsetColumnEnd = 0,
     sourceName,
```

Comment:
Does not set default value to `offsetLineEnd` and `offsetLineEnd`, it's default value should be same as `offsetLine` and `offsetColumn`